README.md

```diff
@@ -13,9 +13,50 @@ You will find the following directories here:
 * `metadata/` - containing static metadata for the addons in `addons/`
 * `test/` - containing integration tests for the addons in `addons/`
 
+# Support
+
+There is **no community support for this software at this time**, and this repository (while publicly visible) is only intended for consumption by D2iQ personnel.
+
+If you are here looking for addon support as an end-user, support is given as per the support provided for your [D2iQ Ksphere Solution](https://d2iq.com/solutions/ksphere)'s provided [enterprise support](https://d2iq.com/services-and-support) (solutions such as [Konvoy](https://d2iq.com/solutions/ksphere/konvoy)), but **support is not given *directly* via this repository**.
+
+Please [contact D2iQ](https://d2iq.com/contact) for questions and more information.
+
+# Releases
+
+Releases are tags which are cut from `master` and other branches and are intended to signify a single supported instance of `kubernetes-base-addons`.
+
+While you may see other tags in our [releases page](https://github.com/mesosphere/kubernetes-base-addons/releases) the only releases which are official and supported releases are those designated as non-prerelease and specifically mentioned to be official releases.
+
+**WARNING**: Do not use `master` for production, instead pick a supported release version.
+
+## Creating a Release
+
+Creating a releases is as simple as cutting a tag and making a [Github release](https://help.github.com/en/github/administering-a-repository/creating-releases).
+
+If the release is meant to be an official KSphere supported release, ensure it's not marked as a pre-release, follows the `v{MAJOR}.{MINOR}.{PATCH}` [semver](https://semver.org/) pattern, and ensure the language in the title and description indicate "Official Release".
```

Comment:
Don't we want `v{MAJOR}.{MINOR}.{PATCH}`version to be `stable-v{MAJOR}.{MINOR}.{PATCH}-#`?

Reply:
cc @jieyu @jimmidyson

Reply:
I've updated it to simply use that pattern instead of straight semver, as we don't have any other use cases right now :+1: